Source/Script/PowerShell/Tfs/Archive.ps1

```diff
@@ -0,0 +1,16 @@
+﻿Param
+(
+    $FtpUser,
+    [SecureString]
+    $FtpPassword,
+    $FtpHost
+)
+
+mkdir $(Agent.BuildDirectory)\zip;
+
+Add-Type -Assembly 'System.IO.Compression.FileSystem';
+
+[System.IO.Compression.ZipFile]::CreateFromDirectory("$(Agent.BuildDirectory)\artifacts\drop\", "$(Agent.BuildDirectory)\zip\$(Build.BuildNumber).zip\");
```

Comment:
build number, I assume is from the build system is not great, it should use the information from git to determine build description